### PR TITLE
Jetpack: update single product cards

### PR DESCRIPTION
**Pull request by keoshi**
#### Changes proposed in this Pull Request

* Update copy for single products.
* Hide promo nudge for products without discounted prices.
* Reordered the cards to show Backup, Search, Scan.

#### Testing instructions

* Fire up this PR.
* Go to WordPress.com and open Plans (`/plans/`) with a Jetpack site.
* Ensure all single product cards have:
  * The correct copy on each.
  * That the promo nudge only appears for Search.
  * Visually look good. For this one, bear in mind the product variation design refresh is being worked on https://github.com/Automattic/wp-calypso/pull/40583

#### Before

image

#### After

image

Fixes https://github.com/Automattic/wp-calypso/issues/40551

### CODE REVIEW

File: client/lib/products-values/constants.js [hunk 1]
```diff
@@ -157,10 +157,11 @@ export const getJetpackProductsTaglines = () => {
 
 export const getJetpackProductsDescriptions = () => {
 	const searchDescription = translate(
-		'Enhanced Search for more relevant results using modern ranking algorithms, ' +
-			'boosting of specific results, advanced filtering and faceting, and more.'
+		'Incredibly powerful and customizable, Jetpack Search helps your visitors instantly find the right content – right when they need it.'
```
Comment (a8ci18n) [inline client/lib/products-values/constants.js]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Fully customizable. Incredibly powerful. Jetpack Search helps your visitors find the right content – right when they need it.' )` *ES Score: 9*

File: client/lib/products-values/constants.js [hunk 2]
```diff
@@ -200,34 +202,17 @@ export const getJetpackProducts = () => {
...
-			slugs: JETPACK_SCAN_PRODUCTS,
-		} );
 	isEnabled( 'jetpack/search-product' ) &&
 		output.push( {
 			title: translate( 'Jetpack Search' ),
-			// TODO: Add new description copy for Search
 			description: translate(
-				'Always-on backups ensure you never lose your site. Choose from real-time or daily backups.'
+				'Incredibly powerful and customizable, Jetpack Search helps your visitors instantly find the right content – right when they need it.'
 			),
```
Comment (jsnmoon) [inline client/lib/products-values/constants.js]: How about this instead?

```jsx
			description: getJetpackProductsDescriptions()[ PRODUCT_JETPACK_SEARCH ],
```

File: client/lib/products-values/constants.js [hunk 3]
```diff
@@ -283,6 +268,23 @@ export const getJetpackProducts = () => {
 			},
 			slugs: JETPACK_SEARCH_PRODUCTS,
 		} );
+	isEnabled( 'jetpack/scan-product' ) &&
+		output.push( {
+			title: translate( 'Jetpack Scan' ),
+			description:
+				'Automatic scanning and one-click fixes keep your site one step ahead of security threats.',
```
Comment (jsnmoon) [inline client/lib/products-values/constants.js]:
```suggestion
			description: getJetpackProductsDescriptions()[ PRODUCT_JETPACK_SCAN ],
```

File: client/lib/products-values/constants.js (hunk 2 above)
Comment (keoshi) [reply client/lib/products-values/constants.js:210]: Yes, of course! That makes a lot of sense.

File: client/lib/products-values/constants.js (hunk 3 above)
Comment (keoshi) [reply client/lib/products-values/constants.js:275]: Yep, great catch — especially since it wasn't even translating the description.

**Pull request merged**